CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Pulumiverse
+
+:+1::tada: Thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing. These are just
+guidelines, not rules, so use your best judgment and feel free to propose
+changes to this document in a pull request.
+
+## Issues and Pull Requests
+
+- If you're not sure about adding something, open an issue to discuss it.
+- Feel free to open a Pull Request early so that a discussion can be had as
+  changes are developed.
+
+## Commit Messages and Pull Request Titles
+
+We use the
+[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+specification to standardize our commit history. To enforce this convention on
+commit messages and/or pull request titles, we use the
+[Semantic Pull Requests](https://github.com/probot/semantic-pull-requests) bot.
+
+The commit message summary (or pull request title) is constructed by prepending
+the type of change being made (e.g., feat, fix, refactor), followed by an
+imperative, present tense sentence (without a period). Example:
+`fix: make header bold`.
+
+If you are still working on your pull request, prepend `WIP:` to indicate that
```

Comment:
I hinted at the Draft pull request feature, why the need for the `WIP:` prefix then?

Reply:
What are you saying?

Are you saying that you rather have what you mentioned above than what you have marked here?

Reply:
Github created the Draft Pull Request feature exactly for this purpose, so I do not want to put the burden upon contributors to remember to add the `WIP:` prefix. As a consequence, my proposal is to drop the requirement to use this prefix from the contribution guidelines.

Reply:
Both options are open for the contributor. They can either use `WIP:` or the draft function.